lib/modules/manager/sveltos/readme.md

```diff
@@ -0,0 +1,39 @@
+The [Sveltos](https://projectsveltos.github.io/sveltos/) manager allows to manage the dependencies of Helm-Charts for Sveltos resources. Read more about Sveltos Helm-Charts in the [Sveltos documentation](https://projectsveltos.github.io/sveltos/addons/helm_charts/).
+
+### Set `fileMatch` pattern
```

Comment:
```suggestion
### You must set a `fileMatch` pattern
```

Put the key info in the heading: the reader must do something.

Reply:
we've other managers with empty file match, we should align them to use same layout